api/src/main/java/org/openmrs/scheduler/SchedulerUtil.java

```diff
@@ -30,20 +30,7 @@
 	 * @param p properties used to start the service
 	 */
 	public static void startup(Properties p) {
-		// Override the Scheduler constants if specified by the user
```

Comment:
@dilekamadushan please remove any change made to `SchedulerUtil` as @wluyima mentioned

Reply:
@teleivo ok I will do the necessary modifications. Thanks